### PR TITLE
change classname to titlecase#268

### PR DESCRIPTION
**Pull request by jayantsaraf**

### CODE REVIEW

**Comment by abhsag24** [comment]
Hi thank you for your Patch make sure you have the right commit structure that is 
```
tag: Shortlog

body
Fixes <issue number/url>
```
Also please upload a screenshot if you have made changes to the gui.

File: build/lib/uiplib/scheduler.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
+"""Module that schedules the wallpaper change."""
```
Comment (abhsag24) [inline build/lib/uiplib/scheduler.py]: why is this file here?

File: uiplib/scheduler.py [hunk 2]
```diff
@@ -17,7 +17,7 @@
 from threading import Thread
 
 
-class scheduler:
+class Scheduler:
```
Comment (nkprince007) [inline uiplib/scheduler.py]: This isn't the only change. You've to change all other instances where this class is being used.

Comment (jayantsaraf) [inline uiplib/scheduler.py]: can you please point out another instance where this class is used?

**Comment by nkprince007** [comment]
And you've messed up the comments again..!

File: uiplib/scheduler.py (hunk 2 above)
Comment (nemaniarjun) [inline uiplib/scheduler.py]: Use git grep

Comment (abhsag24) [inline uiplib/scheduler.py]: @jayantsaraf u can also use `ctrl + shift + f` on some editors or even github project search :P

**Comment by abhsag24** [comment]
issue already solved, closing this PR
